unit-tests/live/image-quality/test-basic-depth.py

```diff
@@ -26,41 +23,87 @@
 
     pipeline = rs.pipeline(ctx)
     cfg = rs.config()
-    cfg.enable_stream(rs.stream.depth, 848, 480, rs.format.z16, 30)
+    cfg.enable_stream(rs.stream.depth, 640, 480, rs.format.z16, 30)
+    cfg.enable_stream(rs.stream.color, 640, 480, rs.format.bgr8, 30) # needed for finding the ArUco markers
     profile = pipeline.start(cfg)
-    frames = pipeline.wait_for_frames()
     time.sleep(2)
 
     depth_sensor = profile.get_device().first_depth_sensor()
     depth_scale = depth_sensor.get_depth_scale()
 
-    depth_passes = {name: 0 for name in depth_points}
 
+    # find region of interest (page) and get the transformation matrix
+    M, _ = find_roi_location(pipeline, (4,5,6,7), DEBUG_MODE)  # markers in the lab are 4,5,6,7
+    depth_passes = {}
     for i in range(NUM_FRAMES):
         frames = pipeline.wait_for_frames()
         depth_frame = frames.get_depth_frame()
+        color_image = frames.get_color_frame()
         if not depth_frame:
             continue
 
         depth_image = np.asanyarray(depth_frame.get_data())
+        color_image = np.asanyarray(color_image.get_data())
+        colorizer = rs.colorizer()
+        colorized_depth_image = np.asanyarray(colorizer.colorize(depth_frame).get_data())
+
+        # use M to get the region of interest - our colored grid printed in the lab
+        a3_img = cv2.warpPerspective(depth_image, M, (A4_WIDTH, A4_HEIGHT))
 
+        # Known pixel positions and expected depth values (in meters)
+        # Using temporary values until setup in lab is completed
+        depth_points = {
+            "cube":  ((a3_img.shape[0] // 2, a3_img.shape[1] // 2), 0.45), # center of page, cube at 0.45m
```

Comment:
We use A4 constants in the helper function but here we say A3.
Please align or make it a parameter we can choose from